.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!-- Before creating a bug report, try disabling browser extensions to see if the bug is still present. -->
+
+<!-- If you're having trouble updating your profile, it is likely because you logged in separately with GitHub & Twitter. Please check if this is the case before creating a bug report, and email yo@dev.to so we can merge your accounts. -->
```

Comment:
Maybe remove this line?

Reply:
c&p... :D removed! (but I'm allowed to do so, no worries)